message_definitions/v1.0/common.xml

```diff
@@ -873,6 +882,26 @@
         <param index="6">Longitude</param>
         <param index="7">Altitude (ground level)</param>
       </entry>
+      <entry value="86" name="MAV_CMD_NAV_DELAY_BUTTON">
+        <description>Wait for a button state change. This is intended to be used as a method to allow a takeoff delay using a custom button.</description>
+        <param index="1">button bitmask</param>
+        <param index="2">Empty</param>
```

Comment:
Shouldn't there be a mask of what button state is desired? As is it reads as accept any edge change, where most use cases I can see are either "wait for rising/falling edge" or "be in the pressed/depressed state" which isn't covered by the message.

Reply:
In practice it is much better to wait for any edge. I previously had it wait for a state, but it is unreliable if the transition is short, as button library is polling